mantidimaging/gui/windows/main/view.py

```diff
@@ -354,3 +355,23 @@ def set_images_in_stack(self, uuid: UUID, images: Images):
 
     def find_images_stack_title(self, images: Images) -> str:
         return self.presenter.get_stack_with_images(images).name
+
+    def dragEnterEvent(self, event: QDragEnterEvent):
+        if event.mimeData().hasUrls():
+            event.acceptProposedAction()
+
+    def dropEvent(self, event: QDropEvent):
+        for url in event.mimeData().urls():
+            file_path = url.toLocalFile()
+            if not os.path.exists(file_path):
+                continue
+            if os.path.isdir(file_path):
+                # Load directory as stack
+                sample_loading = self.presenter.load_stacks_from_folder(file_path)
+                if not sample_loading:
+                    QMessageBox.critical(
+                        self, "Load not possible!", "Please provide a directory that has .tif or .tiff files in it, or "
+                        "a sub directory that do not contain dark, flat, or 180 in their title name, that represents a"
+                        " sample.")
+            else:
+                QMessageBox.critical(self, "Load not possible!", "Please drag and drop only folders/directories!")
```

Comment:
If I select multiple images from a folder and drag them, then I get the "Please drag and drop only folders/directories!" for each one. Could that be set to only show once per drop.

Reply:
I will find a fix! good spot!